frontend/express/public/javascripts/countly/vue/components/date.js

```diff
@@ -935,11 +945,24 @@
                 );
             },
             handleConfirmClick: function() {
+                if (this.rangeMode === 'inBetween') {
+                    var _minDate = new Date(this.minDate).getTime();
+                    var _maxDate = new Date(this.maxDate);
```

Comment:
Shouldn't it also have .getTime()?
I tried on your server - it is object. So check in line 952 will not work.